eclipse-theia/theia
# Correctly localize debug schema attributes

#### What it does

Closes https://github.com/eclipse-theia/theia/issues/13004

Changes the behavior of the debug contributions so that the configurations are filled on the frontend instead of the backend. That allows access to the user locale to translate any description values.

Also eliminates a lot of dead/duplicated code in `vscode-debug-adapter-contribution`. The file appears to be unused and I don't expect any adopters to make use of the file.

#### How to test

1. Install a language pack and switch to the new language
2. Hover over the "builtin" attributes of the debug launch configs, such as `type` or `name`.
3. They should be correctly localized to the current language.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
